televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/kotlin/viewmodels/DetailsViewModel.kt

```diff
@@ -205,23 +212,21 @@ class DetailsViewModel(theMovieDBService: TheMovieDBService, justWatchAPIService
         val year = calcUtils?.getYear(selectedSearchType)
         val seasonsCount = calcUtils?.numberOfSeasons()
 
-        detailsItemViewModel = DetailsItemViewModel(
-                episodeCount,
-                runtime,
-                bingeTime,
-                imageUrl,
-                thumbnailUrl,
-                castResponse,
-                streamDetails,
-                year,
-                category,
-                showTitle.get().toString(),
-                tvShowByIdResponse.episodeDescription,
-                seasonsCount,
-                tvShowByIdResponse)
-
-
-        return detailsItemViewModel as DetailsItemViewModel
+
+        return DetailsItemViewModel(
```

Comment:
change this to a data class as its just for holding data

pros u get with data class:
1. clear error messages when asserting in unit tests along with easily create test data fixtures
2.get to use ready made equals/hashcode and copy functions

Reply:
Fixed